contracts/farms/common/RevoFarmBot.sol

```diff
@@ -88,39 +96,22 @@ contract UbeswapFarmBot is ERC20, AccessControl, Pausable {
 
     address public reserveAddress;
 
-    modifier ensure(uint256 deadline) {
-        require(deadline >= block.timestamp, "FarmBot: EXPIRED");
-        _;
-    }
-
     constructor(
         address _owner,
         address _reserveAddress,
-        address _stakingRewards,
         address _stakingToken,
         address _revoFees,
-        address _swapRouter,
-        address _liquidityRouter,
         address[] memory _rewardsTokens,
```

Comment:
a little weird to me that rewardsTokens would stay, but the stakingRewards contract is removed. seems like they should go together

Reply:
I agree, it is weird that they're not together here. The reason for this was that we can't really make any assumptions about the staking rewards contract, but we _do_ make the assumption that rewards are paid out as some number of ERC20 tokens, so I codified that in the base-most class. I've removed that req though, so now subclasses are free to handle it as they please.